docs/authentication/saml/authentication-flows.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: SAML SSO authentication flows
+title: SAML SSO (enterprise connections) authentication flows
```

Comment:
How about keeping it as it was? SAML already implies enterprise, as we're also mentioning the new flow below.

Reply:
that's fine